pkg/model/network.go

```diff
@@ -164,6 +164,25 @@ func (b *NetworkModelBuilder) Build(c *fi.ModelBuilderContext) error {
 				RouteTable:      publicRouteTable,
 				InternetGateway: igw,
 			})
+
+			for i, route := range b.Cluster.Spec.Topology.AdditionalRoutes {
+				if route.InternetGateway != nil {
+					ig := &awstasks.InternetGateway{
+						Name:      route.InternetGateway,
+						Lifecycle: b.Lifecycle,
+						VPC:       b.LinkToVPC(),
+						Shared:    fi.Bool(true),
+					}
+					c.AddTask(ig)
+					c.AddTask(&awstasks.Route{
+						Name:            s(fmt.Sprintf("public-%v-0.0.0.0/0", i)),
```

Comment:
You might want to require the additional routes to have a `Name`, just so that you can then use it here.  `i` isn't going to be very stable (e.g. if you insert a Route as the first in the list)